src/components/AddContent/AddContent.tsx

```diff
@@ -155,7 +155,7 @@ export const AddContent = ({ isLoading }: Props) => {
     return { distributionArches, distributionVersions };
   }, [distArches, distVersions]);
 
-  const handleModalToggle = () => setIsModalOpen((prev) => !prev);
+  const handleModalToggle = () => setIsModalOpen(!isModalOpen);
```

Comment:
This fixes a react error where the initial time you click the addContent button it would throw an error in the console (as "prev" doesn't exist yet)